overview.md

```diff
@@ -0,0 +1,161 @@
+# SourceCred
+
+SourceCred is anchored by a simple goal: we want open-source contributors to
+earn a living, and a share in the value created on top of their contributions.
+At first, we focused on the obvious problem: where will the money come from? We
+were inspired by the ability of cryptocurrencies to fund open-source projects,
+and started [SourceGrain], with the goal of letting every open-source project
+issue its own token.
+
+However, as this idea evolved, we ran into a big problem: we don't know who
+deserves credit in an open-source project, and thus we don't know how to
+distribute the rewards. The obvious approaches, like counting commits or lines
+of code, would be far too easy to game—and would in any case miss out on vital
+work like design or issue triaging.
+
+This inability to quantify contributions accurately affects every method for
+funding open-source. Projects need a fair system for deciding how to distribute
+rewards regardless of whether they come through sponsorships, donations,
+[cryptocurrency airdrops], or through any [other means]. SourceCred is designed
+to do just that.
```

Comment:
and we hope it will be used downstream by many sources for funding and analyzing open-source contributions.